packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.mdx

```diff
@@ -3,6 +3,7 @@ import * as menuButtonColumnStories from './table-column-menu-button.stories';
 import ComponentApisLink from '../../../docs/component-apis-link.mdx';
```

Comment:
This is unrelated to this PR but while testing it I noticed a bit of clipping at the left edge of the menu button text (only for "Yellow" since the Y seems to go further left than other letters). I only see this in Firefox, not Edge or Safari.

Happy to file a bug if you / @fredvisser think it's worth fixing but don't want to just knock it out.

![image](https://github.com/user-attachments/assets/c6c98dd8-3bbd-45e0-a172-630a57a07a14)

Reply:
I can't see that behavior on Firefox (on Windows), so I'm wondering if it's a Firefox on Mac issue only. Since I can't reproduce it, I don't plan to knock it out as part of this PR. Filing a bug probably makes the most sense.

Reply:
#2332